frontend/package.json

```diff
@@ -43,6 +43,6 @@
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.4",
-    "vite": "^4.5.0"
+    "vite": "^5.0.0"
```

Comment:
It will as long as we have node 18 / 20+ [[source](https://vitejs.dev/guide/migration#node-js-support)]